GEECS-PythonAPI/geecs_python_api/controls/devices/geecs_device.py

```diff
@@ -503,7 +459,56 @@ def _process_scan(cmd: str, comment: str = 'no scan', monitoring_device: Optiona
                     pass
 
         return next_folder, next_scan, accepted, timed_out
+        
+    def update_ini_file(ini_file_path, comment):
```

Comment:
Recommend marking this a `@staticmethod`. Good for clarity, but also that way geecs_device_instance.update_ini_file(path, comment) will work correctly.

Reply:
Will do. Admittedly, I’m not too knowledgeable about static vs non static

Reply:
Nice improvement. Recommend adding a check that ini_file_path is indeed a text file and not something dramatic like "C:/Program Files". I believe the "else" case would erase it all.

Reply:
I think it's safe, this part should catch any issues i think:
        # Extract number from the filename
        match = re.search(r'ScanInfoScan(\d{3})\.ini$', str(ini_file_path))
        if match:
            scan_number = int(match.group(1))
        else:
            print("Filename does not match expected format. Aborting update.")
            return

also, os.remove wouldn't delete directories either. 

I'll go ahead and merge with master and delete the branch